app/src/components/layout.css

```diff
@@ -0,0 +1,628 @@
+/**
+ * Global style defaults
+ *
+ * Adapted from:
+ * https://github.com/gatsbyjs/gatsby-starter-default/
```

Comment:
This file provides boilerplate CSS defaults from Gatsby. We don't necessarily have to include it.